getting-started/adding-telerik-ui-to-an-application/installation-adding-application-namespace-declaration.md

```diff
@@ -5,60 +5,93 @@ description: Namespace Declaration
           	
-#### __XAML__
+	#### __[XAML] Example 1: Telerik namespace definition__ 
+	{{region installation-adding-application-namespace-declaration-0}}
+		xmlns:telerik="http://schemas.telerik.com/2008/xaml/presentation"
+	{{endregion}}
 
-{{region installation-adding-application-namespace-declaration_0}}
-	xmlns:telerik="http://schemas.telerik.com/2008/xaml/presentation"
-{{endregion}}
+	Or select it from the IntelliSense in XAML.
 
-Or you can select it from the IntelliSense in XAML.![Common Installing Namespace Declaration 012](images/Common_InstallingNamespaceDeclaration_012.png)
+	{% if site.site_name == 'WPF' %}
+	![Common Installing Namespace Declaration 1](images/installation-adding-application-using-intellisense-vs-wpf-0.png)
+	{% else %}
+	![Common Installing Namespace Declaration 2](images/installation-adding-application-using-intellisense-vs-wpf-1.png)
+	{% endif %}
 
 * The result should be similar to this.
 
-#### __XAML__
-
-{{region installation-adding-application-namespace-declaration_1}}
+	{% if site.site_name == 'WPF' %}
+	#### __[XAML] Example 2: Telerik namespace added in XAML__
+	{{region installation-adding-application-namespace-declaration-1}}
+		<Window x:Class="WpfApplication1.MainWindow"
+				xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+				xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+				xmlns:telerik="http://schemas.telerik.com/2008/xaml/presentation"
+				Title="MainWindow" Height="350" Width="525">
+			<Grid>
+			</Grid>
+		</Window>
+	{{endregion}}
+	{% else %}
+	#### __[XAML] Example 2: Telerik namespace added in XAML__
+	{{region installation-adding-application-namespace-declaration-2}}
 	<UserControl x:Class="SilverlightApplication1.MainPage"
 	    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 	    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 	    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	    xmlns:telerik="http://schemas.telerik.com/2008/xaml/presentation"
 	    mc:Ignorable="d"
-	    d:DesignHeight="300" d:DesignWidth="400">
-	
-	    <Grid x:Name="LayoutRoot" Background="White">
-	        
+	    d:DesignHeight="300" d:DesignWidth="400">	
+	    <Grid x:Name="LayoutRoot" Background="White">	        
 	    </Grid>
 	</UserControl>
-{{endregion}}
-
-* Now you should be able to declare any controls existing in this namespace and which is in the assemblies you have added in your project. An example of __RadTreeView__ declaration is shown below.
-
-#### __XAML__
-
-{{region installation-adding-application-namespace-declaration_2}}
+	{{endregion}}
+	{% endif %}
+
+* Now you should be able to declare any controls existing in this namespace and which is in the assemblies you have added in your project. An example of RadTreeView declaration is shown below.
```

Comment:
We can improve the wording here:
"...existing in the telerik namespace which are included in the assemblies you have added to your project."